### DISCUSSION TITLE
Git-bash Window Default settings problem

### DISCUSSION THREAD

**Discussion by ghost** [git-for-windows/git#4015] [Q&A] [answered]
Edit: How do i leave this discussion?? I'm stuck with it on my front page

**Comment by rimrul** [comment, marked as answer]
If you installed with the MinTTY console option(the default):

Check your `$HOME` directory for a file called `.minttyrc`. Delete the lines `Rows=1900` and `Columns=1200` from it. That should fix it.

If you installed with the Conhost console option, I don't know how to fix this.

**Reply by dscho** [reply]
(I am also assuming that MinTTY is used and not ConHost, @ZascuOfficial look at `C:\Program Files\Git\etc\install-options.txt` to find out.)

It is probably a good idea to configure logging (i.e. add `Log file` and `Logging` statements to `.minttyrc`, see [the documentation for more details](http://mintty.github.io/mintty.1.html#CONFIGURATION)).

**Reply by dscho** [reply]
And yes, the most likely explanation why Git Bash does not work anymore is that MinTTY refers to the width and height in terms of character columns and character rows, not in terms of pixels. With that in mind, 1900 columns and 1200 rows would translate to something like 17,100 x 20,000 pixels with the default Lucida 9pt font, which seems a bit excessive.

**Reply by ghost** [reply]
How do I leave this discussion? I am stuck with it on my recent repositories panel and it's annoying.
